src/main/scala/org/broadinstitute/dsde/firecloud/webservice/MethodsApiService.scala

```diff
@@ -1,102 +1,120 @@
 package org.broadinstitute.dsde.firecloud.webservice
 
-import akka.actor.{Actor, Props}
+import akka.actor.Props
 import org.broadinstitute.dsde.firecloud.FireCloudConfig
 import org.broadinstitute.dsde.firecloud.core.{AgoraPermissionActor, AgoraPermissionHandler}
 import org.broadinstitute.dsde.firecloud.model.MethodRepository._
 import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol._
 import org.broadinstitute.dsde.firecloud.service.{FireCloudDirectives, PerRequestCreator}
-import org.slf4j.LoggerFactory
-import spray.http.StatusCodes
+import spray.http.{HttpMethods, Uri}
 import spray.httpx.SprayJsonSupport._
 import spray.json.DefaultJsonProtocol._
 import spray.routing.{HttpService, Route}
 
-class MethodsApiServiceActor extends Actor with MethodsApiService {
-  def actorRefFactory = context
-  def receive = runRoute(routes)
-}
-
-object MethodsApiService {
+trait MethodsApiServiceUrls {
   val remoteMethodsPath = FireCloudConfig.Agora.authPrefix + "/methods"
   val remoteMethodsUrl = FireCloudConfig.Agora.baseUrl + remoteMethodsPath
   val remoteConfigurationsPath = FireCloudConfig.Agora.authPrefix + "/configurations"
   val remoteConfigurationsUrl = FireCloudConfig.Agora.baseUrl + remoteConfigurationsPath
   val remotePermissionsTemplate = FireCloudConfig.Agora.baseUrl + FireCloudConfig.Agora.authPrefix + "/%s/%s/%s/%s/permissions"
   val remoteMultiPermissionsUrl = remoteMethodsUrl + "/permissions"
+
+  val localMethodsPath = "methods"
+  val localConfigsPath = "configurations"
```

Comment:
moved urls + paths into a trait to make it easier to mix in elsewhere, and eliminated the companion object to avoid confusion.